### PR TITLE
Authorization and Authentication Pages

### PR DESCRIPTION
**Pull request by bajajcodes**
- Login and Signup pages supports responsiveness, on 768 pixels wide breakpoint reach media or images display get set to none and form takes the full place.
- Minimial design, consistency and clean ui is approach for pages design.

https://user-images.githubusercontent.com/29247011/154841077-3f62aace-7140-4784-a0a1-940cb0c04ffd.mp4

### CODE REVIEW

**Review by nikhilmalviya-dev — APPROVED** [review]
code is fine

File: css/auth.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+}
+
+.input{
+    cursor: initial;
+    padding: var(--size-7) var(--size-9);
+    border: 0.0625rem solid;
+    border-radius: var(--size-3);
+}
+
+.tandc{
```
Comment (Hamdrive) [inline css/auth.css]: Consider going with better and meaningful classname

**Review by Hamdrive — COMMENTED** [review]
Code written looks neat, good use of utility classes overall
Have made some comments to use better class names, apart from that rest all looks good.

**Review by RakshithBhat03 — APPROVED** [review]
Code looks good and has good use of variables.

File: css/auth.css (hunk 1 above)
Comment (bajajcodes) [reply css/auth.css]: I changed class name to `signup_conditions`.

**Pull request merged**